src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,116 @@
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 50px 0;
+  margin: 0 auto;
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  color: var(--text-color-main-accent);
+  border: 1px solid var(--card-border-color);
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  margin: 0 24px;
+
+  &:nth-child(4n) {
+    margin-bottom: 46px;
+  }
+
+  &:nth-last-child(-n+4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__title {
+    margin: 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__subtitle {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--text-color-secondary);
+    margin-bottom: 16px;
+  }
+
+  &__rating-inner {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__rating-info {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    align-self: end;
+  }
+
+  &__price-inner {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: var(--text-color-secondary);
+  }
+
+  &__price-cost {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__button {
+    text-transform: uppercase;
+    width: 100%;
+    height: 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--card-button-color);
+    color: var(--text-color-button);
+    text-decoration: none;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    border: 1px solid transparent;
+    transition: all linear 0.2s;
+  }
+
+  &__button:hover {
```

Comment:
```suggestion
  &:hover {
```
move it to `&__button`

Reply:
<img width="938" alt="image" src="https://user-images.githubusercontent.com/102411130/219303800-8ede274d-5d6c-4859-a9ff-e425bfd9bad2.png">